doc/user_guide/install.rst

```diff
@@ -67,18 +68,28 @@ manager:
 This will install also install the optional GUI packages ``hyperspy_gui_ipywidgets``
 and ``hyperspy_gui_traitsui``. To install hyperspy without the GUI packages, use:
 
-
    .. code-block:: bash
 
        $ conda install hyperspy-base -c conda-forge
 
+.. note::
+
+    Depending on how Anaconda have been installed, it is possible that the 
+    ``conda`` is not avaible from the Terminal, read the
+    `Anaconda User Guide <https://docs.continuum.io/anaconda/>`_ for details.
+
 .. note::
 
     Using ``-c conda-forge`` is only necessary when the conda-forge is not
-    already added to the conda configuration, see the
+    already added to the conda configuration, read the
     `conda-forge documentation <https://conda-forge.org/docs/user/introduction.html>`_
     for more details.
 
+.. note::
+
+    Depending on the packages installed in Anaconda, ``conda`` can be slow and
+    in this case ``mamba`` can be used as replacement of ``conda``. Read the
+    `mamba documentation <https://github.com/mamba-org/mamba>`_ for instructions.
 
 Further information
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
a solution for dependencies resolution -> dependencies
In such situation -> In such a situation